### PR TITLE
C++ tutorial

### PR DESCRIPTION
**Pull request by ErvinaDe**
Added a C++ tutorial

## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s) | Remove Resource(s) | Add info | Improve Repo

### Checklist:
- [ ] Not a duplicate
- [ ] Included author(s) if appropriate
- [ ] Lists are in alphabetical order
- [ ] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

**Comment by ErvinaDe** [comment]
I made sure to follow the guidelines.

File: free-courses-en.md [hunk 1]
```diff
@@ -125,7 +125,7 @@
 
 
 ### C++
-
```
Comment (SethiPandi) [inline free-courses-en.md:128]: Please add this line break back.

**Review by SethiPandi — COMMENTED** [review]
There should be a line break after a heading.

**Comment by SethiPandi** [comment]
I'm closing this as the resource is already in the repository.

**Pull request closed without merging**